### PR TITLE
feat: add Language and parsingInstruction to LlamaParseReader

### PR DESCRIPTION
**Pull request by InsightByAI**
Implemented Language and parsingInstruction to LlamaParseReader and added one example.

A bit hard to test as you can't really get any info from the jobs running on LlamaParse, but adding the LaTeX instruction, see llamaparse_2, did work as expected.
Second try because I'm an idiot and left my API key in the example lol

### CODE REVIEW

**Comment by InsightByAI** [comment]
The example is more or less the implementation of this [Blog Post](https://www.llamaindex.ai/blog/launching-the-first-genai-native-document-parsing-platform) from LlamaIndex

File: packages/core/src/readers/LlamaParseReader.ts [hunk 1]
```diff
@@ -58,7 +63,7 @@ export class LlamaParseReader implements FileReader {
     let response = await fetch(url, {
       signal: AbortSignal.timeout(this.maxTimeout * 1000),
       method: "POST",
-      body,
+      body: body,
```
Comment (himself65) [inline packages/core/src/readers/LlamaParseReader.ts:66]:
```suggestion
      body,
```

File: examples/readers/src/llamaparse_2.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import fs from "fs";
```
Comment (himself65) [inline examples/readers/src/llamaparse_2.ts:1]: use `fs/promises`

File: examples/readers/src/llamaparse_2.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  const reader = new LlamaParseReader({
+    resultType: "markdown",
+    language: "en",
+    parsingInstruction:
+      "The provided document is a manga comic book. Most pages do NOT have title. It does not contain tables. Try to reconstruct the dialogue happening in a cohesive way. Output any math equation in LATEX markdown (between $$)",
+  });
+  const documents = await reader.loadData("../data/manga.pdf"); // The manga.pdf in the data folder is just a copy of the TOS, due to copyright laws. You have to place your own. I used "The Manga Guide to Calculus" by Hiroyuki Kojima
+
+  // Assuming documents contain an array of pages or sections
+  const parsedManga = documents.map((page) => page.text).join("\n\n");
```
Comment (himself65) [inline examples/readers/src/llamaparse_2.ts:15]: it's good to separate pages using `---`

**Review by himself65 — CHANGES_REQUESTED** [review]
LGTM, but need some small change

**Review by himself65 — APPROVED** [review]

**Comment by himself65** [comment]
Btw, if you leak api key, please revoke it because commit could be found even you force push

**Comment by InsightByAI** [comment]
> Btw, if you leak api key, please revoke it because commit could be found even you force push 

Yeah that was the first thing I did lol. Thanks tho!

**Pull request merged**